### DISCUSSION TITLE
Can I use custom color module symbols by specifying a nerd-font icon?

### DISCUSSION THREAD

**Discussion by SherLock707** [fastfetch-cli/fastfetch#1411] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/65c98ee9-d12b-4df7-bef1-aad3bf8723bd)

Wondering if I can use any font icon to get custom color symbols here.

**Comment by CarterLi** [comment, marked as answer]
Use custom module to print what you like.

**Reply by SherLock707** [reply]
Thanks for the quick response and solution!

I was trying it with custom module like:


```
{
      "key": "{#white}│  colors │ ",
      "type": "custom",
      "keyColor": "white",
      "format": "󰚑 󰚑 󰚑 󰚑 󰚑 󰚑 󰚑 󰚑"
  }
```
![image](https://github.com/user-attachments/assets/8ff6af02-2a90-4aff-a96b-03ecd24434a7)


Not sure why there are two rows..

**Reply by CarterLi** [reply]
Try set `"format": ""` to the next custom entry.

**Reply by SherLock707** [reply]
Cool! worked! thanks.

**Comment by SherLock707** [comment]
![image](https://github.com/user-attachments/assets/1472c18a-5b44-415a-bb2a-9d0dddffd338)

Thanks!
